TeacherWorkout.Api/appsettings.CI.json

```diff
@@ -7,6 +7,25 @@
     }
   },
   "ConnectionStrings": {
-    "TeacherWorkoutContext": "host=localhost;Port=5432;Database=teacher_workout_test;User Id=docker;Password=docker;"
-  }
+    "TeacherWorkoutContext": "host=localhost;Port=5432;Database=teacher_workout_test;User Id=docker;Password=docker;",
+    "IdentityDbConnectionString": "host=localhost;Port=5432;Database=teacher_workout_test;User Id=docker;Password=docker;"
+  },
+  "JwtConfig": {
+    "Secret": "",
+    "TokenExpirationInSeconds": 3600
+  },
+  "PasswordOptions": {
+    "RequiredLength": 6,
+    "RequiredUniqueChars": 0,
+    "RequireNonAlphanumeric": false,
+    "RequireLowercase": false,
+    "RequireUppercase": false,
+    "RequireDigit": false
+  },
+  "Admin": {
+    "EmailAddress": "",
+    "Password": ""
+  },
+  "SeedUsers": false,
```

Comment:
Why do we need the separation between Users and Data to seed ? I'd assume we just run the seed and it will take care of everything. If we do, please update the Readme to reference how we seed all the data (imagine you are cloning the project and trying to set it up from scratch).